### PR TITLE
feat(apiaccess): add methods for managing insights insert keys

### PR DESCRIPTION
**Pull request by ctrombley**

### CODE REVIEW

File: pkg/apiaccess/insights_keys.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import "fmt"
+
+type InsightsKey struct {
+	ID        int    `json:"id"`
+	AccountID int    `json:"account_id"`
+	Key       string `json:"key"`
+	Notes     string `json:"notes"`
+	IsEnabled bool   `json:"is_enabled"`
+	//CreatedAt time.Time `json:"created_at"`
+	//UpdatedAt time.Time `json:"updated_at"`
```
Comment (sanderblue) [inline pkg/apiaccess/insights_keys.go:12]: Left commented on purpose?

Comment (ctrombley) [reply pkg/apiaccess/insights_keys.go:12]: Nope!  Good catch

Comment (ctrombley) [reply pkg/apiaccess/insights_keys.go]: Fixed

**Review by sanderblue — APPROVED** [review]
🐑 it

**Review by zlesnr — APPROVED** [review]

**Pull request merged**